includes/class-convertkit-settings-restrict-content.php

```diff
@@ -100,6 +113,7 @@ public function get_by_key( $key ) {
 	public function get_defaults() {
 
 		$defaults = array(
+			'enabled'                => '',
```

Comment:
Shouldn't the default be true or on?

Reply:
No - as discussed with @brendancarney, this defaults to disabled, requiring users to enable/opt in to use this functionality, given the new APIs used and potential for changes to the functionality in future updates.

At this stage we want this to be a feature that users opt in to use.